src/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.o
```

Comment:
What's this separate .gitignore file and did you need to move these ignores to the main repo .gitignore?